ed25519-dalek/Cargo.toml

```
@@ -56,7 +56,7 @@ hex = "0.4"
 bincode = "1.0"
 serde_json = "1.0"
 criterion = { version = "0.5", features = ["html_reports"] }
-hex-literal = "0.4"
+hex-literal = "1.0"
```

This avoids dependabot trying to constantly bump the minor version:

```suggestion
hex-literal = "1"
```